pyriemann/sampling.py

```diff
@@ -0,0 +1,325 @@
+import numpy as np
+from functools import partial
+from pyriemann.utils.base import sqrtm
+from sklearn.utils import check_random_state
+
+
+def _pdf_r(r, sigma):
+    """pdf for the log of eigenvalues of a SPD matrix
+
+    Probability density function for the logarithm of the eigenvalues of a SPD
+    matrix samples from the Riemannian Gaussian distribution [1]_.
+
+    Parameters
+    ----------
+    r : ndarray shape (n_dim,)
+        vector defines in R^n_dim
+    sigma : float
+        dispersion of the Riemannian Gaussian distribution
+
+    Returns
+    -------
+    p : float
+        pdf applied to data point r
+
+    Notes
+    -----
+    .. versionadded:: 0.2.8.dev
+
+    References
+    ----------
+    .. [1] S. Said, L. Bombrun, Y. Berthoumieu, and J. Manton, “Riemannian
+        Gaussian distributions on the space of symmetric positive definite
+        matrices”, IEEE Trans Inf Theory, vol. 63, pp. 2153–2170, 2017.
+        https://arxiv.org/pdf/1507.01760.pdf
```

Comment:
In fact, Notes and References should be added only to public functions.
You should remove this from this function, and add Notes to all new public functions and References to all new public functions making reference to Said207.